server/util/networking/networking.go

```diff
@@ -280,6 +284,22 @@ func (a *HostNetAllocator) unlock(netIdx int) {
 	a.inUse[netIdx] = false
 }
 
+// Returns whether the xt_comment kernel module is available.
+var xtCommentLoaded = sync.OnceValue(func() bool {
+	b, err := exec.Command("lsmod").CombinedOutput()
+	if err != nil {
+		log.Warningf("lsmod failed: %s", err)
```

Comment:
is this going to be noisy on mac, or is this code not included?

Reply:
this code should only be included on linux - going to close this PR though because I think Vadim's suggestion of using a chain might be a better way of doing it since it doesn't depend on this xt_comment module being available